src/pages/discovery/loadingAnimation/loadingAnimation.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import "./loadingAnimation.scss";
+import loadingAnimation from "../../../assets/loadingAnimationAssets/loadingAnimation.svg";
+
+/**
+ * Renders a centered loading animation when active. To use, initialize with 'loading' prop, and set to true for display,
+ * and false for none.
+ * @returns {JSX.Element}
+ * @constructor
+ */
+const LoadingAnimation = (props) => {
```

Comment:
Specify the props. See the proptypes pattern used in the other components as an example.